curriculum/js1/2.js

```diff
@@ -7,7 +7,15 @@
  */
 
 const solution = (a, b, c) => {
-  return 0;
+  if(a > b && a > c){
+    return a;
+  };
+  if(b > a && b > c){
```

Comment:
did you mean.... `return b`?

Reply:
It passed all the tests. I will fix it from my end :) I see the mistake.

Reply:
Tahkns!